### DISCUSSION TITLE
is it necessary to set the minSdkVersion at 24?

### DISCUSSION THREAD

**Discussion by pierroo** [hyochan/react-native-iap#2423] [Q&A]
> Please use the [Discussion board](https://github.com/dooboolab-community/react-native-iap/discussions) if you want to get some help. Please use issues to report bugs.

**Description**

The documentation says we should set the minSdkVersion to 24.
is that mandatory for it to work?
Lots of people are still on older android devices, would it work on a minSdkVersion 21?

**Expected Behavior**

Lots of people are still on older android devices, would it work on a minSdkVersion 21?

**Comment by andresesfm** [comment]
I think the minSdk version was set in accordance to the react-native version. but you are right, technically the library could be used with older versions of Android. Please let us know if you test with a lower version and we can adjust that value.

**Reply by pierroo** [reply]
I have tested wiuth a lower version of android, back to minsdk 21 and so far it seems to work fine
